monocle/worker.py

```diff
@@ -897,6 +921,32 @@ def smart_throttle(self, requests=1):
         except (TypeError, KeyError):
             return False
 
+    async def add_lure_pokestop(self, pokestop):
```

Comment:
the point behind this is that this would be "taking" money from niantic, which may even be, depending on the country something that could be considered a crime